apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -172,7 +173,23 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
             readingSubmissionDate,
         ] = map(row, 'value')
 
-        addons.valuesAmount = [value1, value2, value3, value4].filter(Boolean).length
+        const addons = {
+            address: null,
+            unitType: null,
+            propertyId: null,
+            propertyMap: null,
+            meterId: null,
+            meterResourceId: null,
+            readingSubmissionDate: null,
+            invalidReadingSubmissionDate: null,
+            valuesAmount: 0,
+            value1: normalizeMeterValue(value1),
```

Comment:
Should this helper return null as in columns above?

Reply:
If `valueX` already exists and it will be return `null`, then `valueX` will be rewrite to `null` from some value. But in this case you should not change `valueX`. 
Therefore, `valueX` with `null` is invalid. And empty line is simply ignored.